src/org/opendatakit/briefcase/ui/reused/AggregateServerConnectionConfiguration.java

```diff
@@ -0,0 +1,68 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import org.opendatakit.briefcase.reused.BriefcaseException;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Objects;
+
+public class AggregateServerConnectionConfiguration {
+  private URL url;
+  private String username;
```

Comment:
The username and password are not required. You should implement a `Credentials` ValueObject class to store them and this class should have an `Optional<Credentials>` member instead of username and password